modules/dpu-install-topology.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/dpu-hardware-offload.adoc
+
+:_content-type: CONCEPT
+[id="openshift-and-dpu-architecture_{context}"]
+= OpenShift and DPU deployment architecture
+
+To ease the management of DPUs, the architecture is a two-cluster design. In this architecture, DPU cards are provisioned as worker nodes of the ARM-based infrastructure cluster. The tenant cluster composed of the x86 servers is where the normal user applications run. The following diagram illustrates the deployment architecture.
```

Comment:
"ease the management of DPUs" -> I don't think that's the reason from a customer's perspective. The real reason for the two-cluster design is that as of today, you can't run aarch64 and x86 in the same cluster.

Reply:
@bn222 excellent point